server/protocols/jmap-rfc-8621-integration-tests/postgres-jmap-rfc-8621-integration-tests/src/test/java/org/apache/james/jmap/rfc8621/postgres/PostgresThreadGetTest.java

```diff
@@ -88,31 +60,11 @@ public class PostgresThreadGetTest extends PostgresBase implements ThreadGetCont
             .overrideWith(new TestJMAPServerModule()))
         .build();
 
-    @AfterEach
-    void tearDown() throws IOException {
-        client.close();
-    }
-
     @Override
     public void awaitMessageCount(List<MailboxId> mailboxIds, SearchQuery query, long messageCount) {
-        awaitForOpenSearch(queryConverter.from(mailboxIds, query), messageCount);
     }
 
     @Override
     public void initOpenSearchClient() {
-        client = MailboxIndexCreationUtil.prepareDefaultClient(
-            openSearch.getDockerOpenSearch().clientProvider().get(),
-            openSearch.getDockerOpenSearch().configuration());
-    }
-
-    private void awaitForOpenSearch(Query query, long totalHits) {
```

Comment:
Why remove this?

Reply:
In postgres implementation, Thread feature use postgres instead of opensearch